sys/androidmedia/gstamc.h

```diff
@@ -72,6 +72,7 @@ struct _GstAmcCodec
   guint flush_id;
   GMutex buffers_lock;
   gboolean tunneled_playback_enabled;
+  gboolean enable_adaptive_playback;
```

Comment:
this field doesn't seem to be used anywhere. We initialize it to TRUE in gst_amc_codec_new , but never use it.